tests/unit/OptionsListUtilsTest.js

```diff
@@ -199,17 +199,17 @@ describe('OptionsListUtils', () => {
         // When we call getNewChatOptions() with no search value
         let results = OptionsListUtils.getNewChatOptions(REPORTS, PERSONAL_DETAILS, '');
 
-        // Then no reports should be returned, only personalDetails and all the personalDetails should be returned
-        // minus the currently logged in user
-        expect(results.recentReports.length).toBe(0);
-        expect(results.personalDetails.length).toBe(_.size(PERSONAL_DETAILS) - 1);
+        // We should expect maxmimum of 5 recent reports to be returned
+        // minus the currently logged in user and recent reports count
```

Comment:
Not sure that this is correct. 

> We should expect maxmimum of 5 **_recent reports_** to be returned minus the currently logged in user and **_recent reports_** count

Reply:
I think this is two different things, no?
- Recent reports should have a max of 5
- `personalDetails` --> all the personalDetails should be returned minus the currently logged in user and recent reports count

Reply:
I got inspired by [getNewGroupOptions()](https://github.com/Expensify/Expensify.cash/blob/cb4c331ba5b09fcf4695faac9c2e6ff83b499366/tests/unit/OptionsListUtilsTest.js#L265-L270) test and I think this is correct

Reply:
I think it was previously two different comments that have been merged into one?

Reply:
The code is fine

Reply:
What I meant here is that there should be two separate comments, but I think you have combined them into one.
```
// We should expect maxmimum of 5 recent reports to be returned --> this is referring to `recentReports`
// minus the currently logged in user and recent reports count --> why would the `recentReports` count be `X - 5 - logged in user`? 
```

I think it should say this: 
```
// We should expect maxmimum of 5 recent reports to be returned
// We should expect all personalDetails to be returned, minus the currently logged in user and recent reports count
```

Does that make sense?

Reply:
Yes, I refactored the commenting